### ISSUE TITLE
Video in portrait mode is cropped

### ISSUE THREAD

**Issue by wnm** [biati-digital/glightbox#196]
Hi,

When I embed a vimeo video that is in potrait mode, like https://vimeo.com/496705079, it crops it up:

![Screenshot_2021-01-13 Asana Hawaii](https://user-images.githubusercontent.com/705094/104439547-7ff03680-5591-11eb-84dd-797f0cd7dec9.png)

Is there there an option that I'm missing, or is that an css issue?

**Comment by wnm** [comment]
I think it's because glightbox is setting the plyr ratio option to 16:9

If I start plyr with the ratio option, it crops the video too:
https://codepen.io/wnm-the-lessful/pen/YzGJMpv

If I start plyr without the opition it loads the player in the ratio the video is in:
https://codepen.io/wnm-the-lessful/pen/poExBRj

Any way I can remove the ratio option in glightbox? I tried setting it to null or an empty string, but both didn't work.

**Comment by biati-digital** [comment]
That's weird, setting the value to empty should be enough, did you set your value like this?

```javascript
const lightbox = GLightbox({
  plyr: {
    config: {
      ratio: '',
      muted: false,
      hideControls: true,
    }
  }
});
```

If all the videos are vertical in the meantime you can try to set the ratio to `ratio: '4:6'`

**Comment by wnm** [comment]
hi, thanks for your reply!

setting the value to an empty string, does indeed work to let plyr figure out the right proportions based on the video. 
It chooses the correct proportions for the vimeo video, but the video is still cropped, now probably due to some css rules?

![Screenshot_2021-01-21 TEST](https://user-images.githubusercontent.com/705094/105366141-4659a400-5bff-11eb-8187-f084daeaf715.png)

It looks like there is an max-width set, but not a max-height?

**Comment by biati-digital** [comment]
I'll have to make some tests to find out what's happening, currently I don't have the time, probably till the end of February.

**Comment by biati-digital** [comment]
Fixed in 3.0.8, if you leave the ratio empty Plyr will calculate the correct size and GLightbox will use it and resize it correctly. Please note that Plyr takes a few seconds to calculate the size, there's nothing we can do to speed up the process.
